### ISSUE TITLE
Function 'f' has no '__code__' member (no-member)

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#213]
Originally reported by: **Antony Lee (BitBucket: [anntzer](http://bitbucket.org/anntzer), GitHub: @anntzer?)**

---

The following snippet triggers the spurious error given in the title.

```
#!python

def f():
    pass
print(f.__code__)
```

Other attributes, e.g. `__closure__`, trigger the same error.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/213
